### ISSUE TITLE
GeoBrowser does not display kmz file

### ISSUE THREAD

**Issue by emanueldima** [clarin-eric/switchboard-tool-registry#95]
@fungunga For this kmz file: https://infra.clarin.eu/content/VLO_KML/language_sites.kmz the GeoBrowser(KML) does not show anything: no loaded file and also no error. 

Is this related to the size of the input file? In any case, an error message is necessary.

**Comment by fungunga** [comment]
Geo-Browser can only display certain KML formats, please see here: <https://geobrowser.de.dariah.eu/beta/doc/geobrowser.html#specification-for-use>

Error handling has to be enhanced, yes: <https://projects.gwdg.de/projects/geo-browser/work_packages/34360>

**Comment by emanueldima** [comment]
The pdf specifies the following attributes: 
<img width="370" alt="Screenshot 2020-12-10 at 11 34 01" src="https://user-images.githubusercontent.com/231453/101760829-b0007180-3adb-11eb-8f6f-abe72fa272c4.png">
 And the document given as example has these attributes (the first element as example):
````
            <Placemark>
                <name>!Xóõ</name>
                <description>genus: Southern Khoisan&lt;br&gt;family: Khoisan&lt;br&gt;&lt;br&gt;&lt;a href='https://infra.clarin.eu/content/language_info/data/nmn.html'&gt;More information...&lt;/a&gt;</description>
                <LookAt>
                    <longitude>21.5</longitude>
                    <latitude>-24</latitude>
                    <altitude>0</altitude>
                    <heading>-0.1094821371621389</heading>
                    <tilt>0</tilt>
                    <range>204302</range>
                    <altitudeMode>relativeToGround</altitudeMode>
                </LookAt>
                <styleUrl>#msn_ylw-pushpin1</styleUrl>
                <Point>
                    <coordinates>21.5,-24,0</coordinates>
                </Point>
            </Placemark>
````
So is it the lack of a `TimeStamp` or `Link` that is causing the failure? Or the presence of additional elements?

**Comment by fungunga** [comment]
I do not really know how this is handled internally, but you can look into the existing examples under “static data“: <https://geobrowser.de.dariah.eu/data/boell.kml>, error handling sure has to be improved.

**Comment by fungunga** [comment]
A direct Geo-Browser call <https://geobrowser.de.dariah.eu/?kml1=https://trep.de.dariah.eu/1.0/dhcrud/hdl:21.T11991/0000-001B-4228-C/data> does work just fine.

A Switchboard call with the same object (https://trep.de.dariah.eu/1.0/dhcrud/hdl:21.T11991/0000-001B-4228-C/data) <https://switchboard.clarin-dev.eu/switchboard-test/#/trep/https%3A%2F%2Ftrep.de.dariah.eu%2F1.0%2Fdhcrud%2Fhdl:21.T11991%2F0000-001B-4228-C%2Fdata/application%2Fvnd.dariahde.geobrowser.kml%2Bxml> links to <https://geobrowser.de.dariah.eu/?kml=https%3A%2F%2Fswitchboard.clarin-dev.eu%2Fswitchboard-test%2Fapi%2Fstorage%2F7cae1a3b-33e2-4f65-b4c3-78f242268cdf%3Fmediatype%3Dapplication%252Fvnd.google-earth.kml%252Bxml>, and it does not work.

BTW: Direct calls of <https://switchboard.clarin-dev.eu/switchboard-test/api/storage/7cae1a3b-33e2-4f65-b4c3-78f242268cdf?mediatype%3Dapplication%252Fvnd.google-earth.kml%252Bxml> and <https://switchboard.clarin-dev.eu/switchboard-test/api/storage/7cae1a3b-33e2-4f65-b4c3-78f242268cdf> are working just fine...

And finally <https://geobrowser.de.dariah.eu/?kml=https%3A%2F%2Fswitchboard.clarin-dev.eu%2Fswitchboard-test%2Fapi%2Fstorage%2F7cae1a3b-33e2-4f65-b4c3-78f242268cdf> also is working just fine!

Puuh! Sorry for all the URLs :-) Can we prevent the mimetype to be forwarded to the Geo-Browser parameter? Or would that be an Geo-Browser issue?

**Comment by emanueldima** [comment]
@fungunga I am looking at the problems you signalled, but in my opinion the bigger usability problem is a lack of an error message when the GeoBrowser cannot show anything due to missing fields. Can you do something about that?

**Comment by fungunga** [comment]
Yes, this already is an issue to be fixed: anhancing error handling in geobrowser: https://projects.gwdg.de/projects/geo-browser/work_packages/34360/activity?query_id=505
